README.md

```diff
@@ -1,753 +1,283 @@
-<img src="static/logo.png" width="400" />
```

Comment:
No more logo? :(

Reply:
Yes you're right marketing and branding should probably decide on this.
If you do remove it, you might want to remove it from `static/logo.png` too.